testing/functional/tests/cypress/fixtures/draftApplicationData.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file is missing formatting and has too many blank lines.

Reply:
In the latest you will find the code in formatted and at the end of lines ";" has been added